drivers/dai/intel/ssp/ssp.h

```diff
@@ -235,6 +235,11 @@
 #define SHIM_CLKCTL_I2SFDCGB(x)		BIT(20 + x)
 #define SHIM_CLKCTL_I2SEFDCGB(x)	BIT(18 + x)
 
+/* Is dynamic clock gating a must? */
+#if DT_PROP(DT_NODELABEL(ssp_settings), force_dynamic_clock_gating)
```

Comment:
We need to stop using hard-coded nodelabels and replace them with phandle references.